docs/README.md

```diff
@@ -38,12 +38,12 @@ The Measurement Plug-In SDK for LabVIEW packages enable measurement developers t
 
 1. Create and save a new LabVIEW project.
 
-2. From the project window, go to `Tools` → `Plug-In SDKs` → `Create Measurement Plug-in...`.
+2. From the project window, go to `Tools` → `Plug-Ins` → `Measurement` → `Create Measurement Plug-in...`.
     - In the dialog, enter the `Measurement Plug-in Name` and `Measurement Plug-in Description`, then click `Create Measurement Plug-in`.
 
         ![New measurement dialog](images/New%20measurement%20dialog.png)
 
-    - This will create a new measurement service library.
+    - This will create a new measurement service library and a UI library
```

Comment:
We lost the period that ends this sentence; probably best include it, as that is consistent with all the other line items here.